src/application/clients/crux-gateway-bridge.ts

```diff
@@ -0,0 +1,104 @@
+import * as net from "net";
```

Comment:
Leave this file out, we will add it later when we actually add auth logic

Reply:
changed